client/src/components/Tea/TeaEditor/TeaEditorContainer.jsx

```diff
@@ -115,7 +115,9 @@ export class TeaEditorContainer extends React.Component {
   };
 
   getBrandsFromTeas = teas => {
-    return teas.map(tea => tea.brand);
+    return Object.keys(teas).map(teaId => {
```

Comment:
`Object.keys` is not recommended when you could otherwise store the list of teas as an array. If you need to access teas individually and in a list, store both an array and object of teas in your state. `keys().map()` is an O(2n) operation - twice as inefficient as it needs to be.

Taking this a step further, you could memoize this result (using [reselect](https://github.com/reduxjs/reselect) or otherwise) so you don't have to iterate twice, or even once, upon every render call.

Reply:
Does this function still work now that `teas` has been refactored?